lib/mongo_client.js

```diff
@@ -110,6 +112,7 @@ const validOptions = require('./operations/mongo_client_ops').validOptions;
  * @param {boolean} [options.monitorCommands=false] Enable command monitoring for this client
  * @param {number} [options.minSize] If present, the connection pool will be initialized with minSize connections, and will never dip below minSize connections
  * @param {boolean} [options.useNewUrlParser=false] Determines whether or not to use the new url parser
+ * @param {string} [options.optionsValidationLevel] Sets the optionsValidationLevel for options validation: 'none', 'warn', or 'error'
```

Comment:
Let's try not to use the name of the property to recursively define the property. "Sets the output level for options validation, one of: 'none', 'warn', or 'error'."

Also, indicate that the default is `warn`